tests/std/lit.site.cfg.in

```diff
@@ -1,18 +1,42 @@
 # Copyright (c) Microsoft Corporation.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-config.cxx_archive_root            = "@CMAKE_ARCHIVE_OUTPUT_DIRECTORY@"
-config.cxx_headers                 = "@STL_TESTED_HEADERS_DIR@"
-config.cxx_library_root            = "@CMAKE_LIBRARY_OUTPUT_DIRECTORY@"
-config.cxx_runtime_root            = "@CMAKE_RUNTIME_OUTPUT_DIRECTORY@"
-config.expected_results_list_path  = "@STD_EXPECTED_RESULTS@"
-config.include_dirs                = ["@LIBCXX_SOURCE_DIR@/test/support", "@STL_SOURCE_DIR@/tests/std/include"]
-config.msvc_toolset_libs_root      = "@TOOLSET_LIB@"
-config.stl_build_root              = "@STL_BUILD_ROOT@"
-config.stl_src_root                = "@STL_SOURCE_DIR@"
-config.target_arch                 = "@VCLIBS_TARGET_ARCHITECTURE@"
-config.test_exec_root              = "@STD_TEST_OUTPUT_DIR@"
-config.test_subdirs_file           = "@STD_TEST_SUBDIRS_FILE@"
-config.test_subdirs_root           = "@STD_TEST_SUBDIRS_ROOT@"
+import os
+import site
+site.addsitedir("@STL_TEST_UTILS_DIR@")
+site.addsitedir(os.path.join("@LIBCXX_SOURCE_DIR@", "utils"))
+import stl.test.config
+import stl.test.features
+import stl.test.file_parsing
+import stl.test.format
+import stl.test.params
 
-lit_config.load_config(config, "@CMAKE_CURRENT_SOURCE_DIR@/lit.cfg")
+config.name           = 'std'
+config.suffixes       = ['test[.]cpp$', 'test[.]compile[.]pass[.]cpp$']
+config.test_exec_root = "@STD_TEST_OUTPUT_DIR@"
+config.test_format    = stl.test.format.STLTestFormat()
+

```

Comment:
Things should go in the config file if the information is needed to actually run the test. Otherwise you should attempt to keep the information in the lit_config. Configs are embedded in test objects which are pickled and sent to worker processes to execute. Keeping the test object small do the pickling and transmission is fast is important in my experience.